modules/light-clients/08-wasm/types/vm.go

```diff
@@ -108,7 +108,7 @@ func wasmCall[T ContractResult](ctx sdk.Context, clientStore storetypes.KVStore,
 
 	resp, err := callContract(ctx, clientStore, cs.CodeHash, encodedData)
 	if err != nil {
-		return result, errorsmod.Wrapf(err, "call to wasm contract failed")
+		return result, errorsmod.Wrapf(ErrWasmContractCallFailed, err.Error())
```

Comment:
is `Wrapf` needed when not providing a templated string? I see we're using `Wrap` above

Reply:
yup, noticed this too but there's so many instances of this I think it should a separate PR as to not make this one more taxing to review.

Reply:
https://github.com/cosmos/ibc-go/pull/4985